<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>361</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>621</width>
    <height>361</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>621</width>
    <height>361</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QFrame" name="openImgFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 144, 144);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="findFileButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>15</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>파일 찾기</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileLocationLadel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>15</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>다음 &gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="sampleImgFrame">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(202, 202, 202);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
